.editorconfig

```diff
@@ -9,10 +9,11 @@ insert_final_newline = true
 end_of_line = lf
 
 [*.{kt, kts}]
+ktlint_code_style = intellij_idea
```

Comment:
same behavior?

Reply:
[See](https://pinterest.github.io/ktlint/latest/rules/code-styles/#:~:text=Starting%20from%20version%201.0%2C%20ktlint_official%20is%20the%20default%20code%20style) and [this](https://github.com/pinterest/ktlint/pull/2144/files#diff-4e705beb17bb15a0aafaa58ead16c9ec09bd02cf2c8a2fc2c14fb375a87ccab6L54) which landed in 1.0. This was causing all the problems in https://github.com/cashapp/paparazzi/pull/1183